docs/windows.md

```diff
@@ -130,7 +132,7 @@ kube-proxy version.
 
 ```bash
 # Example:
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.18.0/g'  > kube-proxy.yml
+curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/kube-proxy.yml | sed 's/VERSION/v1.18.0/g' > kube-proxy.yml
```

Comment:
Is it possible to use a varible for the version? You can use `v0.1.5` as a default value if it is the version we wanted to pin.

Reply:
Updated.